problems/2.functions.js

```diff
@@ -2,16 +2,17 @@
 * @param {string} name - The name of the user to greet
 * @returns {null}
 */
-function logGreeting() {
+function logGreeting(name) {
+  console.log("Hello" + name + "!")
 
 }
 
 /** Returns a custom greeting string in the form "Hello Avery!"
 * @param {string} name - The name of the user to greet
 * @returns {string} - The custom greeting
 */
-function greeting() {
-
+function greeting(name) {
+ return "Hello" + " " + "name" + "!"
```

Comment:
Don't put name as "name". name is your parameter and should be treated as a variable. "name" is string that says "name"